File: Android/doraemonkit/src/main/java/com/didichuxing/doraemonkit/kit/parameter/ram/RamKit.java
package com.didichuxing.doraemonkit.kit.parameter.ram;

import android.content.Context;

import com.didichuxing.doraemonkit.R;
import com.didichuxing.doraemonkit.constant.FragmentIndex;
import com.didichuxing.doraemonkit.kit.AbstractKit;
import com.didichuxing.doraemonkit.kit.Category;

public class RamKit extends AbstractKit {
    @Override
    public int getCategory() {
        return Category.PERFORMANCE;
    }

    @Override
    public int getName() {
        return R.string.dk_frameinfo_ram;
    }

    @Override
    public int getIcon() {
        return R.drawable.dk_ram;
    }

    @Override
    public void onClick(Context context) {
        startUniversalActivity(context,FragmentIndex.FRAGMENT_RAM);
    }

    @Override
    public void onAppInit(Context context) {

    }
}

File: Android/kotlin/doraemonkit/src/main/java/com/didichuxing/doraemonkit/kit/performance/ram/RamKit.kt
package com.didichuxing.doraemonkit.kit.performance.ram

import android.content.Context
import com.didichuxing.doraemonkit.R
import com.didichuxing.doraemonkit.kit.AbstractKit

class RamKit : AbstractKit() {
    override val name: Int
        get() = R.string.dk_frameinfo_ram

    override val icon: Int
        get() = R.mipmap.dk_ram

    override fun onClick(context: Context?) {
        kotlinTip()
    }

    override fun onAppInit(context: Context?) {}
    override val isInnerKit: Boolean
        get() = true

    override fun innerKitId(): String {
        return "dokit_sdk_performance_ck_arm"
    }
}